self-core-impl/src/main/java/com/selfxdsd/core/GithubIssues.java

```diff
@@ -42,6 +42,8 @@
  * @author Mihai Andronache (amihaiemil@gmail.com)
  * @version $Id$
  * @since 0.0.1
+ * @todo #777:60min In GithubIssues#search() text and labels must be url encoded
+ *  when adding them to the search url.
```

Comment:
@criske what is this todo about exactly?

Reply:
@amihaiemil text and labels are not encoded. For example if we search for `hello world`,  `URI.create(...)` will throw exception for illegal character (`space` is illegal). So `hello world` should look like `hello+world` when is added to uri (this can be done by using `URLEnconder.encode`).